cosim/mk/Makefile.fpga

```diff
@@ -28,7 +28,7 @@ reset_pl:
 	echo 'from pynq import PL; PL.reset();' | sudo -E $(PYTHON3)
 
 unpack_bitstream:
-	base64 -d ../$(BASENAME)_bd_1.$(BOARDNAME).tar.xz.b64 | tar xvJ
```

Comment:
It's a bit weird that if I don't unpack the bitstream at the same folder where the bitstream is stored it will not be unpacked properly.

Reply:
Oh what's your proposal? Pass a path to the packed bitstream?

Reply:
With this users should copy their favorite bitstream to fpga/. The bitstream will be unpacked there.

Reply:
I think in previous versions the bitstream is at ./fpga/../

Reply:
Eh, this flow works better for the NFS mount flow.
```
# On x86
vivado $ pack_bitstream
# On PS
fpga $ unpack_bitstream
```

No copying required

Reply:
I'd accept a change where 
BITSTREAM ?= ../$(BASENAME)_bd_1.$(BOARDNAME).tar.xz.b64
and users can override it

Reply:
Looks like the bug is actually in my head not in the Makefile. I have reverted the related commit.